### DISCUSSION TITLE
How to clean out data?

### DISCUSSION THREAD

**Discussion by GrimmiMeloni** [solectrus/solectrus#2019] [Q&A] [answered]
Due to the earlier experiments with MQTTcollector I have some misaligned data in the system.
I already purged data up to a specific date from Influx.
Solectus however still shows the malformed data. I _assume_ it is due to caching in Redis, but I am not 100% positive on this.
What is the easiest way to get rid of the wrong information completely?

**Comment by ledermann** [comment, marked as answer]
Yes, this seems to be the Redis cache. SOLECTRUS caches the results of Influx queries to speed up the display of completed time periods.

To delete the cache, simply run this command (assuming your Redis container is named `solectrus-redis-1`):

```bash
docker exec -it solectrus-redis-1 redis-cli FLUSHALL
```

Another option is to delete the `redis/` folder, but this requires to stop the containers first:

```bash
docker compose down
rm -r ~/solectrus/redis/*
docker compose up
```

Hope this helps.

**Reply by GrimmiMeloni** [reply]
Yep, the redis-cli flush did the trick. Thanks!
